src/style.css

```
@@ -0,0 +1,63 @@
+body {
+  margin: 0;
+}
+
+/* ul,
+li {
+  margin: 0;
+  padding: 0;
+} */
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+}
+
+.nav-list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav-item {
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.nav-item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav-link {
+  text-decoration: none;
+  color: #000;
+  position: relative;
+  font-family: Roboto;
```

Добавляем альтернативные стили, на случай если Гугл фонтс не будет работать
```suggestion
  font-family: Roboto, Arial, sens-serif;
```